src/Components/CaseGrid/CaseThumb.js

```diff
@@ -93,33 +87,61 @@ const Img = styled.div`
   transform: scale(1.001);
 `
 
+const Description = styled.div`
+  font-size: ${props => props.full ? '30px' : '18px'};
+  & > * {
+    margin-top: 10px;
```

Comment:
Again, please use `props => props.theme.spacing(2)` -> 8 px

Reply:
Also, I would add: 
`
   margin-bottom: 0;
`
Since the inherited p margin seems displace the text vertically.